webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTableFormats.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+export const hostsCountFormat = (value, { rowData: { name } }) => (
+  <a href={`hosts?search=model+%3D+"${encodeURI(name)}"`}>{value}</a>
+);
+
+export const actionsFormat = (value, { rowData: { name, id } }) =>
+  (value.includes('delete') ? (
+    <span className="btn btn-sm btn-default">
+      <a
+        data-confirm={__(`Delete ${name}`)}
+        data-id={`aid_models_${id}-${encodeURI(name)}`}
+        rel="nofollow"
+        data-method="delete"
+        href={`/models/${id}-${encodeURI(name)}`}>
+        Delete
+      </a>
+    </span>
+  ) : null);
+
+export const nameFormat = (value, { rowData: { actions, id, name } }) =>
+  (actions.includes('edit') ? (
```

Comment:
does it make sense to have only one function that does a switch statement on actions, and call the relevant function? I would imagine in other pages we will have a lot more actions.

Also... these are probably generic actions? in that case maybe we should put them under the common namespace?

Reply:
Sounds good, I can work on it in this PR or in a future PR.

Reply:
if its easy, i would move it now, if not, future PR is ok.